onboarding-app/SecondViewController.swift

```diff
@@ -3,27 +3,178 @@ import UIKit
 
 class SecondViewController: UIViewController {
     
-    var label: UILabel!
+    var stackView = UIStackView()
+    var closeButton: UIButton!
+    var titleLabel: UILabel!
+    var sectionTitle = UIStackView()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .black
+        view.backgroundColor = .white
+        view.addSubview(stackView)
         
-        label = UILabel()
-        label.translatesAutoresizingMaskIntoConstraints = false
-        label.text = "HELLO WORLD"
-        label.textColor = .white
-        label.font = .systemFont(ofSize: 40)
+        layoutStackView()
+        layoutCloseButton()
+        layoutTitle()
+        layoutCard(
+            imageSystemName: "building.columns.circle.fill",
+            firstLineText: "An account outside Wise",
+            secondLineText: "Send from your bank",
+            onTap: #selector(showCounterView)
+        )
+        layoutSectionTitle()
+        layoutCard(
+            imageSystemName: "eurosign.circle.fill",
+            firstLineText: "4000 EUR available",
+            secondLineText: "Euro"
+        )
+    }
+    
+    func layoutStackView() {
+        stackView.axis = .vertical
+        stackView.spacing = 20
+        stackView.alignment = .leading
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.isLayoutMarginsRelativeArrangement = true
         
-        view.addSubview(label)
-        constraintsInit()
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+        stackView.layoutMargins = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 20)
     }
     
-    func constraintsInit() {
+    func layoutCloseButton() {
+        let colorConfig = UIImage.SymbolConfiguration(hierarchicalColor: .systemGray)
+        let scaleConfig = UIImage.SymbolConfiguration(pointSize: 40)
+        let closeButtonImage = UIImage(systemName: "xmark.circle.fill")?.applyingSymbolConfiguration(colorConfig)?.applyingSymbolConfiguration(scaleConfig)
+        
+        var buttonConfig = UIButton.Configuration.plain()
+        buttonConfig.image = closeButtonImage
+        // TODO: ask why there's a horizontal margin around the image by default. Is this a default UIButton inset thing? Or is it an SFSymbol thing?
```

Comment:
I have a question here: I noticed that there was quite a bit of horizontal space around the button image by default. However, googling tells me that there shouldn't be that space by default. So I'm wondering, is this something from SF Symbols instead?

Reply:
let's discuss visually :D not getting what you are explaining here